### PR TITLE
added docker intergration

### PR DESCRIPTION
**Pull request by mperkins808**
- docker intergration with the backend emulator to improve mobility of the repo
 - fetching of the git log containing each commit message and the SHA
CC
@macite 
@ShaeChristmas 
@Fletch99

### CODE REVIEW

**Comment by mperkins808** [comment]
@Gradox9 @rickydodd

**Comment by Gradox9** [comment]
Hey @mperkins808, you'll have to add Andrew Cain, Shaine Christmas, Matthew Paul Fletcher to the reviewers section once your ready for your pull request to be submitted so that it can be pulled to the main branch.
image

**Comment by mperkins808** [comment]
> Hey @mperkins808, you'll have to add Andrew Cain, Shaine Christmas, Matthew Paul Fletcher to the reviewers section once your ready for your pull request to be submitted so that it can be pulled to the main branch. image

I can do this but the repo has no write protection to main at the moment so won't make a difference.

**Comment by mperkins808** [comment]
@rickydodd If there are no comments I will merge this sometime after trimester 2 has ended and marking has been completed so its set up for next trimester. I will also add write-protection to the main branch.

File: README.md
Comment (ShaeChristmas) [inline README.md:27]: You will need to make sure this file is run through Prettier and Vale before merging

Comment (mperkins808) [reply README.md:27]: Have linted the code and ran `Prettier` and `Vale`

**Review by rickydodd — APPROVED** [review]

**Pull request merged**
